cellxgene_schema_cli/tests/test_utils.py

```diff
@@ -107,3 +109,12 @@ def test_adata_with_canonical_raw_X(self, adata_with_raw):
     def test_adata_with_canonical_X(self, adata_without_raw):
         enforce_canonical_format(adata)
         assert adata_without_raw.X.has_canonical_format is True
+
+
+class TestGetHashDigestColumn:
+    def test_get_hash_digest_column(self, adata_with_raw):
+        hash_digest_column = get_hash_digest_column(adata_with_raw.obs)
+        assert isinstance(hash_digest_column, Series)
+        for val in hash_digest_column:
+            assert isinstance(val, str)
+            assert len(val) == 10  # 5 ASCII chars / 4 bytes, each ID is 8 bytes
```

Comment:
Do you want to instantiate a test df, and test against the expected product?

Reply:
updated!